<launch>
    <node name="image_entropy_node" pkg="image_entropy"
        type="image_entropy_node" output="screen">
    </node>
</launch>
